features/Steps/PlaceSteps.php

```diff
@@ -152,6 +152,18 @@ public function iDeleteThePlaceAt(string $url): void
         $this->theResponseStatusShouldBe(204);
     }
 
+    /**
+     * @When I delete the place at :url then the response status should be :responseStatus
+     */
+    public function iDeleteThePlaceAtThenTheResponseStatusShouldBe(string $url, int $responseStatus): void
```

Comment:
The approach for exceptions or checking bad requests is just to call `And I send a POST request to `, see https://github.com/cultuurnet/udb3-backend/blob/master/features/auth/user-access-token.feature#L47

Otherwise, a method needs to be called for every specific endpoint.